Build.PL

```diff
@@ -39,15 +39,15 @@ my $build = WTSI::DNAP::Utilities::Build->new
                           'IPC::Run'                  => '>= 0.92',
                           'JSON'                      => '>= 2.90',
                           'Log::Log4perl'             => '>= 1.46',
-                          'List::AllUtils'            => '>= 0.09',
+                          'List::MoreUtils'           => '0',
```

Comment:
We need to keep List::AllUtils, some other classes depend on it. Unless, of course, the dependency is changes there as well.

Reply:
Good point. I'd prefer to switch everything than maintain a mixture of the two.

Reply:
Added a commit to that effect